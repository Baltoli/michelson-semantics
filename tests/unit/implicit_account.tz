$Instructions { IMPLICIT_ACCOUNT } ; 
$Stack ListItem(#KeyHash("12345")) ;
