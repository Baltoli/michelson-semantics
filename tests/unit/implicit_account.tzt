code { IMPLICIT_ACCOUNT } ; 
input { Stack_elt key_hash "@Address(12345)" } ;
output { Stack_elt (contract unit) "@Address(12345)" } ;
other_contracts { Elt "@Address(12345)" unit }
