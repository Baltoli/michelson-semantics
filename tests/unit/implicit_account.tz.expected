<generatedTop>
  <k>
    .
  </k>
  <stack>
    #Contract ( #Address ( "12345" ) , unit .AnnotationBottomList .AnnotationBottomList )
  </stack>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
